platform/android/export/export_plugin.cpp

```diff
@@ -2340,6 +2340,7 @@ static bool has_valid_keystore_credentials(String &r_error_str, const String &p_
 }
 
 bool EditorExportPlatformAndroid::has_valid_username_and_password(const Ref<EditorExportPreset> &p_preset, String &r_error) {
+	String keytool = get_keytool_path();
```

Comment:
Can we use `get_keytool_path()` directly in `has_valid_keystore_credentials` instead of passing it as an argument?

Reply:
Not without changing the signature of **has_valid_keystore_credentials** which currently is static, so no access to the member function get_keytool_path() is given.

Reply:
`get_keytool_path()` is also `static` but the issue seems to be because `has_valid_keystore_credentials` is not declared in the header file. Adding that declaration should resolve the access issue.

Reply:
I missed ```get_keytool_path()``` beeing ```static```. There's not even a need for declaring ```has_valid_keystore_credentials```  in the header file. Calling ```EditorExportPlatformAndroid::get_keytool_path()``` (added class specifier) is sufficient.  
I will amend the pull request accordingly.